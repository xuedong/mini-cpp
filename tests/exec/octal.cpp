#include <iostream>

int main() {
  std::cout << 0100 << "\n";
  std::cout << 0777 << "\n";
  std::cout << 0644 << "\n";
  int maxint = 017777777777;
  std::cout << maxint << "\n";
  int x = 037777777777;
  std::cout << x << "\n";
  x = 020000000000;
  std::cout << x << "\n";
}
