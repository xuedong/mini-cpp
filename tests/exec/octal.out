64
511
420
2147483647
-1
-2147483648
